src/main/java/com/elytradev/concrete/inventory/gui/widget/WBar.java

```diff
@@ -110,10 +110,10 @@ public void paintBackground(int x, int y) {
 		//GuiDrawing.drawString(""+inventory.getField(max)+"", x+32, y+9, 0xFF000000);
 	}
 	
-	public static enum Direction {
+	public enum Direction {
 		UP,
 		RIGHT,
 		DOWN,
-		LEFT;
+		LEFT
```

Comment:
These semicolons are intended and separate the enum constant declarations from the (nonexistant) enum body. They're still good practice.